frontend/src/metabase/dashboard/components/Dashboard/Dashboard.styled.jsx

```diff
@@ -138,4 +141,9 @@ export const CardsContainer = styled(FullWidthContainer)`
     css`
       margin-top: ${space(2)};
     `}
+
+  &.${SAVING_DOM_IMAGE_CLASS} ${DashCardRoot} {
```

Comment:
Maybe it is better to put it inside `DashCardRoot`. Otherwise, we prefer not to import directly from `.styled` and instead we export subparts like 
```
export default Object.assign(DashCard, {
  Root: DashCardRoot,
});

```